normal_operation.md

```diff
@@ -4,67 +4,42 @@ css_id: normal_operation
 
-After a tier-1 supplier has delivered new images to the OEM, and the OEM has somehow verified the authenticity and integrity of these images, the OEM SHOULD test new metadata and images before releasing them, in order to ensure that the images work as intended end user vehicles. To do so, It SHOULD use the following steps.
+### Testing metadata and images
 
-First, the OEM SHOULD add these metadata and images to the image repository. It SHOULD also add information about these images to the inventory database, including any dependencies and conflicts between images for different ECUs. Both of these steps are done, to  make  the new metadata and images available to vehicles.
+After the OEM has somehow verified the authenticity and integrity of new metadata and images received from the tier-1 supplier, the OEM SHOULD test both before releasing them to ensure that the images work as intended on end-user vehicles. To do so, It SHOULD use the following steps.
 
-Optionally, if images are encrypted on demand per ECU, then the OEM SHOULD ensure that the director repository somehow has access to the original, unencrypted images, so that automated processes running the director repository are able to encrypt them in the first place. It does not matter how the original, unencrypted images are stored on the director repository. For example, they MAY be stored unencrypted, or they MAY be encrypted using a master key that is known by the automated processes. See the Preparing an ECU for Uptane page for more details.
+First, the OEM SHOULD add these metadata and images to the Image repository. It SHOULD also add information about these images to the inventory database, including any dependencies and conflicts between images for different ECUs. Both of these steps are done to make the new metadata and images available to vehicles.
 
-Second, the OEM SHOULD test the updated metadata and images on reserved vehicles, before releasing them to all vehicles in circulation, so that it can verify whether these images work as intended. To do so, it MAY instruct the director repository to first install the updated images on these reserved vehicles.
+Optionally, if images are encrypted on demand per ECU, then the OEM SHOULD ensure that the Director repository somehow has access to the original, unencrypted images, so that automated processes running the Director repository are able to encrypt them in the first place. It does not matter how the original, unencrypted images are stored on the Director repository. For example, they MAY be stored unencrypted, or they MAY be encrypted using a master key that is known by the automated processes. See the ["Preparing an ECU for Uptane"](https://github.com/uptane/deployment-considerations/blob/master/ecus.md) page on this website for more details.
```

Comment:
Again, I'm not sure about the "somehow". It seems unnecessary.

```suggestion
Optionally, if images are encrypted on demand per ECU, then the OEM SHOULD ensure that the Director repository has access to the original, unencrypted images, so that automated processes running the Director repository are able to encrypt them in the first place. It does not matter how the original, unencrypted images are stored on the Director repository. For example, they MAY be stored unencrypted, or they MAY be encrypted using a master key that is known by the automated processes. See the ["Preparing an ECU for Uptane"](https://github.com/uptane/deployment-considerations/blob/master/ecus.md) page on this website for more details.
```

Reply:
I think the "somehow" here does provide some additional inflection, but I agree that it is probably not necessary.